install

```diff
@@ -2,20 +2,26 @@
 
 cd `dirname $0`
 
-# The clean script removes temporary and junk files that should not be
-# committed to the source code repo.
-if (test -e clean)
-then
-    ./clean
-fi
-
 if [ "$1" = "--skip-types" ]; then
     shift;
     dont_skip_types=0
 else
     dont_skip_types=1
 fi
 
+dev_install=0
```

Comment:
It'd be better to handle setting dev_install the same way that dont_skip_types is. Doesn't matter which way, just consistency helps understanding the code. I personally like the way dev_install is done with its compactness.